Gemfile

```diff
@@ -4,3 +4,27 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in boilerplate.gemspec
 gemspec
+
+group :development, :test do
+  gem 'activerecord-import'
+  gem 'avro', '~> 1.9'
+  gem 'database_cleaner', '~> 1.7'
+  gem 'ddtrace', '~> 0.11'
+  gem 'dogstatsd-ruby', '~> 4.2'
+  gem 'guard', '~> 2'
+  gem 'guard-rspec', '~> 4'
+  gem 'guard-rubocop', '~> 1'
+  gem 'mysql2', '~> 0.5'
+  gem 'pg', '~> 1.1'
+  gem 'rails', '~> 6'
+  gem 'rake', '~> 13'
+  gem 'rspec', '~> 3'
+  gem 'rspec_junit_formatter', '~>0.3'
+  gem 'rspec-rails', '~> 4'
+  gem 'sqlite3', '~> 1.3'
+end
+
+group :lint do
+  gem 'rubocop', '~> 1.22'
+  gem 'rubocop-rspec', '~> 2.0'
+end
```

Comment:
Don't we usually use the gemspec to manage this? Why are we adding these explicitly in the Gemfile?